Update proxy-setup.adoc (#793)
SUSE/doc-susemanager: modules/installation/pages/proxy-setup.adoc

City::
Further information attached to the proxy's certificate.

State::
Further information attached to the proxy's certificate.

Country Code::
In the [guimenu]``country code`` field, enter the country code set during the {productname} installation.
For example, if your proxy is in the US and your {productname} is in DE, enter `DE` for the proxy.
+

> [TIP] .Country Code

Reviewer: ```suggestion
[NOTE]
```

====

> Beware the country code must be made of two upper case letters.

Reviewer: ```suggestion
The country code must be two upper case letters.
```

> For further information on country codes, refer to the online https://www.iso.org/obp/ui/#search[list of alpha-2 codes].

Reviewer: ```suggestion
For a complete list of country codes, see https://www.iso.org/obp/ui/#search.
```
